festipals/src/components/ProfilePage.js

```diff
@@ -7,8 +7,64 @@ export default class ProfilePage extends Component {
       <div>
       <PageHeader icon="fa fa-user" title="Your Profile" description="(Edit or add some imformation about yourself.)"/>
         <div className="container">
-          <h1>Your Profile</h1>
-          </div>
+        <p>First Name: "firstname"</p>
+        <p>Last Name: "lastname"</p>
+        <p>Email: "email@email.com"</p>
+
+        <button type="button" className="btn btn-default btn-block" data-toggle="modal" data-target="#changeNameModal">Change Name</button>
```

Comment:
I know we had this in the mock-up, but does a user have to be able to change his name?